docs/installing/index.md

```diff
@@ -6,61 +6,96 @@ title: Installing
 # Installing Alaveteli
 
 <p class="lead">
-  There are a number of ways to install Alaveteli.
-  We've made an Amazon Machine Image (AMI) so you can quickly deploy on
-  Amazon EC2 (handy if you just want to evaluate it, for example).
-  If you prefer to use your own server, there's an installation script
-  which does most of the work for you, or you can follow the manual
-  installation instructions.
+  There are a number of ways to install Alaveteli, but we recommend
```

Comment:
:tada: much more likely to read this shorter version rather than letting my gaze slide past, also a much clearer call to action